recipe/meta.yaml

```diff
@@ -1,92 +1,100 @@
 {% set name = "go" %}
 {% set version = "1.11.2" %}
-{% set n = 'm2w64-' if win else '' %}
 
 package:
-  name: {{ name|lower }}
+  name: {{ name }}
   version: {{ version }}
 
 source:
   - url: https://storage.googleapis.com/golang/go{{ version }}.src.tar.gz
     sha256: 042fba357210816160341f1002440550e952eb12678f7c9e7e9d389437942550
     folder: go
-  - url: https://storage.googleapis.com/golang/go1.4-bootstrap-20171003.tar.gz
-    sha256: f4ff5b5eb3a3cae1c993723f3eab519c5bae18866b5e5f96fe1102f0cb5c3e52
-    folder: go-bootstrap
 
 build:
-  number: 1000
+  number: 1001
   binary_relocation: False
   detect_binary_files_with_prefix: False
-  force_ignore_keys:  # [win]
-    - c_compiler      # [win]
-    - cxx_compiler    # [win]
+  force_ignore_keys:    # [win]
+    - c_compiler        # [win]
+    - cxx_compiler      # [win]
+    - fortran_compiler  # [win]
   # test data links to these DSOs
   missing_dso_whitelist:
     - /usr/lib/libSystem.B.dylib
     - /usr/lib/libgcc_s.1.dylib
     - libc.so.6
 
-requirements:
-  run:
-    - {{ pin_subpackage(name.lower()+"_linux-64", exact=True) }}  # [linux]
-    - {{ pin_subpackage(name.lower()+"_osx-64", exact=True) }}    # [osx]
-    - {{ pin_subpackage(name.lower()+"_win-64", exact=True) }}    # [win]
-
-test:
-  commands:
-    - go help
-    - go version
-
 outputs:
-  - name: {{ name|lower }}-core
+  - name: {{ name }}                                                  # [cgo]
+    requirements:                                                     # [cgo]
+      run:                                                            # [cgo]
+        - {{ pin_subpackage(name+"_"+target_platform, exact=True) }}  # [cgo]
+    test:                                                             # [cgo]
+      commands:                                                       # [cgo]
+        - go version                                                  # [cgo]
+
+  - name: {{ name }}-{{ cgo_var }}
     script: build-core.sh   # [not win]
     script: build-core.bat  # [win]
     requirements:
       build:
-        {% if not win %}
-        - {{ compiler('c') }}
-        - {{ compiler('cxx') }}
-        - {{ compiler('fortran') }}
-        {% else %}
-        - {{ n }}toolchain
-        {% endif %}
-
-        - perl  # [not win]
+        - {{ compiler('go14') }}
```

Comment:
We used to build `go 1.4` as the bootstrap compiler every time a new version of go came out. Although google only updates that recipe to fix issues with new compilers, we at conda-forge tend to change our baseline more often, e.g. the recent macOS SDK 10.6->10.9 switch required us to [patch](https://github.com/conda-forge/go1.4-bootstrap-feedstock/pull/4#issuecomment-449608036) the bootstrap compiler.

Furthermore, since we are now building two versions of `go` this seems like an easy way to save the extra CI compute cycles.